frontend/src/libs/components/tooltip/tooltip-wrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import 'react-tooltip/dist/react-tooltip.css';
+
+import { type FC } from 'react';
+import { Tooltip as ReactTooltip } from 'react-tooltip';
+
+import { type DataTooltipId } from '~/libs/enums/enums.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { type ValueOf } from '~/libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  id: ValueOf<typeof DataTooltipId>;
+  shouldHideArrow?: boolean;
+  className?: string;
+};
+
+const Tooltip: FC<Properties> = ({ id, shouldHideArrow, className }) => {
```

Comment:
Let's call it the same way as file
```suggestion
const TooltipWrapper: FC<Properties> = ({ id, shouldHideArrow, className }) => {
```

Reply:
done - [b936b37](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/156/commits/b936b37e4137516679eca10da504587f5184dafc)